application/templates/deployment.yaml

```
@@ -74,6 +74,7 @@ spec:
           ]
 {{- end }}
     spec:
+      serviceAccountName: {{ include "application.serviceAccountName" $ }}
```

Same comment as in the other thread.

```suggestion
      automountServiceAccount: {{ $.Values.rbac.enabled }}
      serviceAccountName: {{ include "application.serviceAccountName" $ }}
```